build.gradle

```diff
@@ -41,8 +41,8 @@ dependencies {
   implementation('com.github.ie3-institute:PowerSystemUtils:2.0') {
     exclude group: 'org.apache.logging.log4j'
     exclude group: 'org.slf4j'
-    /* Exclude our own nested dependencies */
     exclude group: 'com.github.ie3-institute'
+    exclude group: 'com.github.johanneshiry', module: 'OSMonaut'
```

Comment:
One more thing... We don't use OSMonaut anymore, if I remember correctly

Reply:
Yes, also regonized that. Should be removed as well